lib/data/passport_data.yml

```diff
@@ -1711,6 +1711,15 @@ st-maarten:
   renewing_old: 8_weeks
   applying: 8_weeks
   replacing: 8_weeks
+st-martin:
+  type: ips_application_1
+  group: ips_documents_group_1
+  app_form: hmpo_2_application_form
+  online_application: true
+  renewing_new: 6_weeks
+  renewing_old: 8_weeks
+  applying: 8_weeks
+  replacing: 8_weeks
```

Comment:
There are a lot of duplicated settings, some of them could be shared between 60 or so countries. I do not now if this is incidental or if there's anything unifying those countries. I also don't know how likely are the rules on the two sides of the island to change (e.g. marriage rules are different already). I'll leave it as it is for now.